backend_py/user_mock/requirements.txt

```diff
@@ -0,0 +1,2 @@
+fastapi
+uvicorn
```

Comment:
Ideally these (and their sub-dependencies) should be pinned using e.g. `poetry` like we do for the primary backend, so we can rebuild without surprise.

Fix it now or later?

Reply:
Indeed they should, and for the ResInsight/user_grid3d_ri component we'll certainly employ Poetry once we get going.
As for the user_mock component, I suggest we revisit it when the structure of `user_grid3d_ri` has been settled. 
In the meantime I'll pin the versions of `fastapi` and `uvicorn` in the `requirements.txt` file for the mock component.